tools/util_linux.go

```diff
@@ -5,10 +5,11 @@ package tools
 /*
 #include <sys/ioctl.h>
 
-#undef BTRFS_IOCTL_MAGIC
-#define BTRFS_IOCTL_MAGIC 0x94
-#undef BTRFS_IOC_CLONE
-#define BTRFS_IOC_CLONE _IOW (BTRFS_IOCTL_MAGIC, 9, int)
+#undef FICLONE
+#define FICLONE		_IOW(0x94, 9, int)
+// copy from https://github.com/torvalds/linux/blob/v5.2/include/uapi/linux/fs.h#L195
+// for older header files.
+// This is eaual to BTRFS_IOC_CLONE. https://github.com/torvalds/linux/blob/v5.2/include/uapi/linux/btrfs.h#L850
```

Comment:
This should probably say "equal".  I'm also not sure we need the references; it's probably fine to just say something like, "This is equal to the older BTRFS_IOC_CLONE value."